### PR TITLE
Changes added in GEOS to assimilate CrIS-FSR observations from NOAA-21

### PR DESCRIPTION
**Pull request by gmao-wgu**
These resource files have been modified to include CrIS-FSR observations from NOAA-21.

### CODE REVIEW

File: GEOSaana_GridComp/GSI_GridComp/etc/gsi.rc.tmpl [hunk 1]
```diff
@@ -172,9 +172,9 @@ OBS_INPUT::
    iasibufr       iasi        metop-c     iasi_metop-c          0.0     3      0        ncep_mtiasi_bufr
    atmsbufr       atms        npp         atms_npp              0.0     1      0        ncep_atms_bufr
    atmsbufr       atms        n20         atms_n20              0.0     1      0        ncep_atms_bufr
-   atmsbufr       atms        n21         atms_n21              0.0     1      0        ncep_atms_bufr
```
Comment (gmao-msienkie) [inline GEOSaana_GridComp/GSI_GridComp/etc/gsi.rc.tmpl:175]: delete the atms_n21 line?

Comment (gmao-wgu) [reply GEOSaana_GridComp/GSI_GridComp/etc/gsi.rc.tmpl:175]: I guess my base line  does not have that atms_n21 line.  I will add that line back.

**Review by gmao-jjin3 — DISMISSED** [review]

**Review by rtodling — APPROVED** [review]
These look good to me. Please notice I move the changes to the sat-db to

https://github.com/GEOS-ESM/GEOS_mksi/pull/4

**Review by rtodling — COMMENTED** [review]
Where the changes for NPP meant here?!

**Comment by gmao-wgu** [comment]
Ricardo, could you please be more specific here?

**Pull request merged**